src/AdvancedHMC.jl

```diff
@@ -48,6 +48,26 @@ include("diagnosis.jl")
 include("sampler.jl")
 export sample
 
+### AD utilities
```

Comment:
Maybe consider moving this block into `contrib/ad.jl` as well.

Reply:
Sure. I will move them.